struct BeerReview
{
	1: string user_id,
	2: string beer_id,
	3: byte rating,
	4: byte srm,
	5: byte body,
	6: byte bitterness,
	7: byte sweetness,
	8: byte aftertaste,
	9: string comments,
	10: double price,
	11: string place,
	12: string size,
	13: string drankwithfood,
	14: bool food_recommended
	
}
